app/assets/stylesheets/components/_navbar.scss

```diff
@@ -1,6 +1,6 @@
 .navbar-lewagon {
   justify-content: space-between;
-  background: white;
+  background: green;
```

Comment:
Hey i really like this idea it's cool!
You should have a look at colors.com they also have nice green!